Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,18 @@
 import React from 'react';
 import './Header.css';
+// import moment from 'moment';
+
+const HeaderTitle = () => {
+    return (
+        <div className='header-title-wrapper'>
+            <div className='header-title'>
+                Lambda School
+                <span className='twitter-handle'> @LambdaSchool • </span>
+                <span className='time-stamp'>26 Jan</span>
```

Comment:
Check out Moment.js - a super quick import that allows the use of 'moment()': real-time time-stamps for your projects!